tests/integration/lifecycle/test_lifecycle.py

```diff
@@ -51,7 +50,7 @@ def test_basic_lifecycle_actions(new_dir, partitions, mocker):
     # See https://gist.github.com/sergiusens/dcae19c301eb59e091f92ab29d7d03fc
 
     # first run
-    # command: pull
```

Comment:
?

Reply:
ruff think that is comment out code that I should remove. While I don't want to disable it nor add `noqa`. I had to change it not looks like.